### ISSUE TITLE
[Bug]: Desktop client not starting since latest update (Mint Mate, flatpak)

### ISSUE THREAD

**Issue by stullekovski** [nextcloud/desktop#6313]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

After flatpak client update the client is not starting anymore.

$ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=nextcloud --file-forwarding com.nextcloud.desktopclient.nextcloud @@u %u @@

Qt: Session management error: Could not open network socket
nextcloud.gui.application: Migrating old config from “/home/kristen/.var/app/com.nextcloud.desktopclient.nextcloud/data/Nextcloud” to “/home/kristen/.var/app/com.nextcloud.desktopclient.nextcloud/config/Nextcloud”
nextcloud.gui.application: Failed to move the old config directory to its new location ( “/home/kristen/.var/app/com.nextcloud.desktopclient.nextcloud/data/Nextcloud” to “/home/kristen/.var/app/com.nextcloud.desktopclient.nextcloud/config/Nextcloud” )
nextcloud.gui.application: Will move the individual files (“Nextcloud2_sync.log”)
nextcloud.gui.application: Fallback move of “Nextcloud2_sync.log” also failed
Unrecognized option ‘%u’
Try ‘nextcloud --help’ for more information

### Steps to reproduce

update to latest client version via flatpak

### Expected behavior

a start of the client

### Which files are affected by this bug

client launcher

### Operating system

Linux

### Which version of the operating system you are running.

Mint Mate 21.2

### Package

Other

### Nextcloud Server version

27.1.5

### Nextcloud Desktop Client version

3.11.0

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Additional info

On Linux Mint Cinnamon the client is starting but the icon is broken. See: https://github.com/nextcloud/desktop/issues/6312

**Comment by gav-** [comment]
Confirmed

Linux Mint 21.2, Mate DE
Nextcloud Desktop 3.11.0 from flatpak

No file sync works, no systray icon, nextcloud process sits in the background chewing %60 of one CPU core

**Comment by ebdavison** [comment]
I have the same error on Fedora 38 with flatpak version 3.11.0:

```
nextcloud.gui.application: Migrating old config from "/home/edavison/.var/app/com.nextcloud.desktopclient.nextcloud/data/Nextcloud" to "/home/edavison/.var/app/com.nextcloud.desktopclient.nextcloud/config/Nextcloud"
nextcloud.gui.application: Failed to move the old config directory to its new location ( "/home/edavison/.var/app/com.nextcloud.desktopclient.nextcloud/data/Nextcloud" to "/home/edavison/.var/app/com.nextcloud.desktopclient.nextcloud/config/Nextcloud" )
nextcloud.gui.application: Will move the individual files ()
...
** (process:31142): WARNING **: 18:51:14.804: Failed to connect to bus: Unable to create socket: Too many open files

** (process:31142): WARNING **: 18:51:14.806: Failed to connect to bus: Unable to create socket: Too many open files

** (process:31142): WARNING **: 18:51:14.808: Failed to connect to bus: Unable to create socket: Too many open files
```

**Comment by ekasprzak** [comment]
Hi, I think it's more related to latest flatpak build and not to v 3.11 or any specific distro/DE. More details here:

https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/issues/140#issuecomment-1879669251

**Comment by gav-** [comment]
Latest flatpak resolves issue for me (still Nextcloud Desktop v3.11, flatpak update released 2024-01-15)

**Comment by ekasprzak** [comment]
> Latest flatpak resolves issue for me (still Nextcloud Desktop v3.11, flatpak update released 2024-01-15)

Yup, I can confirm it's working now.

**Comment by claucambra** [comment]
Glad to read it's been fixed :)
